CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [unreleased]
+### Security
+### Added
+### Changed
+### Removed
+
+## [v0.7.0] - 26/11/2021
+### Security
+- dependencies update ([#6](https://github.com/flatcar-linux/locksmith/pull/6))
+- etcd, dbus and systemd Go dependencies update ([#8](https://github.com/flatcar-linux/locksmith/pull/8))
```

Comment:
I think we could just aggregate into one that all Go dependencies has been updated to latest versions.